ADVANCED.md

```diff
@@ -1383,12 +1383,13 @@ and a suitable `.eslintrc.json`:
       "error",
       { "vars": "all", "args": "after-used", "ignoreRestSiblings": false }
     ],
+    "@typescript-eslint/explicit-function-return-type": "warn",
```

Comment:
Is this where you imagined I would add it?

Reply:
yeah tho i think we should also move this to the basic cheatsheet.. a little bit more complicated operation. 

btw we can also add comments inside github json blocks since it isnt real json :)

Reply:
oh right - I'll do that now!

Reply:
I'm going to move the whole Linting section